### PR TITLE
0.8.5: Use debug_span for quinn connection

### PR DESCRIPTION
**Pull request by philpax**
As requested in https://github.com/quinn-rs/quinn/pull/1351#issuecomment-1222337965. Apologies for the delay. Wasn't sure if I should update quinn-proto as well; can revert that if it's unnecessary.

### CODE REVIEW

**Review by Ralith — APPROVED** [review]

File: quinn-proto/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "quinn-proto"
-version = "0.8.4"
+version = "0.8.5"
```
Comment (Ralith) [inline quinn-proto/Cargo.toml:3]: No need to bump -proto since you've made no changes there.

Comment (philpax) [reply quinn-proto/Cargo.toml]: Gotcha, fixed

**Review by Ralith — APPROVED** [review]

**Comment by philpax** [comment]
Not sure what to do about the lint; I assume you can just ignore that for now?

**Comment by Ralith** [comment]
Yep, not your responsibility. Thanks!

**Comment by djc** [comment]
This has been published.

**Pull request merged**